crates/aptos-rosetta/src/client.rs

```diff
@@ -436,7 +436,7 @@ impl RosettaClient {
         let unsigned_transaction: RawTransaction = bcs::from_bytes(&hex::decode(
             unsigned_response.unsigned_transaction.clone(),
         )?)?;
-        let signing_message = hex::encode(unsigned_transaction.signing_message());
+        let signing_message = hex::encode(unsigned_transaction.signing_message().unwrap());
```

Comment:
You don't want to propagate this error down (which you could since this function returns `anyhow::Result<String>`?

Reply:
Same for the other `unwrap` below...

Reply:
This client is only used for testing, so I'd rather it just crash.